.github/ISSUE_TEMPLATE/doc_issue.yml

```diff
@@ -40,20 +40,23 @@ body:
     id: collection-version
     attributes:
       label: IBM z/OS Ansible core Version
-      description: |
-        Which version of the z/OS Ansible core collection are you using. If you are unsure, run the command: `cat ~/.ansible/collections/ansible_collections/ibm/ibm_zos_core/MANIFEST.json | grep version`
-      multiple: true
+      description: Which version of z/OS Ansible core collection are you using. If you are unsure, review the [documentation](https://ibm.github.io/z_ansible_collections_doc/faqs/faqs.html#how-do-i-update-a-collection-to-the-latest-version). 
+      multiple: false
       options:
-        - v1.0.0
-        - v1.1.0
-        - v1.2.1
-        - v1.3.0
-        - v1.3.1
-        - v1.3.3
-        - v1.3.5
+        - v1.9.0
+        - v1.8.0
+        - v1.7.0
+        - v1.6.0
```

Comment:
I prefer the new order from newest to oldest by descending. Just two personal doubts
Before were include the betas version, wouldn't be nice to keep with that including the 1.6.0-beta at the moment?
Also Is ok to add all that next versions?
The second doubt apply to the the 3 files changes in ISSUE_TEMPLATE.

Reply:
@AndreMarcel99 - I think what you prefer the descending (newest to oldest) is what how it will show in the git issue, this PR is actually correcting the behavior that you are commenting on. GH issues show the selection box as it is ordered, have a look at [here](https://raw.githubusercontent.com/ansible-collections/ibm_zos_core/dev/.github/ISSUE_TEMPLATE/doc_issue.yml) and you will see the order starts with 1.0 to later versions, so we are both in agreement, and this is aligning to your comment.

Reply:
This template is for documentation, our docs really don't align to a version (we have work items to have version support on our docs in the backlog in JIRA (not yet ported to GH), I figured it might be good to have a version to associate to a doc , maybe its better we do use a `x` for the last bit , eg 1.6.x. Our docs are slightly different where with issues related to modules and source we actually do need to include beta's. I could push this out with `x`'s instead.

Reply:
I think its ok to add all the next versions, its more or less future proofing, if we find users are selecting them when there is no version we can remove them, what I find worse that happens is we are not keeping up with updates to the templates hence I added a few more future release versions. Not sure what is worse, not having the templates updated with the most current release or having future releases, I tend to think its worse to not have the templates updated so when someone tries to open an issue, the version is not there.

Reply:
I suppose another option is as part of our release strategy we update the templates, that could solve this problem.